src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -69,7 +69,7 @@ object Region {
 
   private def cutPrefixSuffix(imp: Global#Import, source: SourceFile): (String, List[String]) = {
     val printedImport = source.content.slice(imp.pos.start, imp.pos.end).mkString
-    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*(\w|\d|_|-)+(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
+    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*((\w|\d|_|-)+|(\`.*\`)+)(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
```

Comment:
Shouldn't you also change some code in the function where I posted that evil example directly from Mordor?